pages/iam/how-to/manage-members.mdx

```diff
@@ -0,0 +1,120 @@
+---
+meta:
+  title: How to manage IAM Members
+  description: Instructions for managing Members in Scaleway IAM.
+content:
+  h1: How to manage IAM Members
+  paragraph: Instructions for managing Members in Scaleway IAM.
+dates:
+  validation: 2025-06-02
+  posted: 2025-06-02
+---
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+
+## How to create a Member
+
+1. Click **Identity and Access Management (IAM)** from the top-right of your [Organization Dashboard](https://console.scaleway.com/organization) in the Scaleway console. The **Users** tab of the [Identity and Access Management dashboard](https://console.scaleway.com/iam/users) displays.
+2. Click **+ Create Member**. A pop-up displays.
+3. Enter a username and the email address of the Member you want to add to your Organization.
+    <Message type="important">
+      When you create a Member, a Scaleway account is created for them. They exist only within your Organization. If you delete the Member, their account is also deleted.
+    </Message>
+4. (Optional) Check the box if you want to send a welcome email to the Member.
+5. Add a password.
+    <Message type="note">
+      This step is optional. If you set a password, make sure you keep note of it to share it with the Member later. The password will only be shown once. From their first login, the Member has up to seven days to update their password and comply with this security requirement.
+    </Message>
+6. (Optional) Check the box if you want to send the password to the Member via email.
+7. Click **Create Member**.
+
+If you did not send an invitation email to the Member, make sure you give them their login information.
+
+## How to lock a Member
+
+As an Owner or user with IAM Manager permissions, you can lock a Member anytime.
+
+<Message type="important">
+  Locking is an action that only applies to IAM Members. Once a Member is locked, they cannot log into the Organization, but are not removed from it.
```

Comment:
Here (or wherever you find it convenient), we could say that API keys attached to the Locked user lose their permission too